README.md

```diff
@@ -29,3 +29,12 @@ What is the asymptotic runtime ($\Theta$) of your algorithm in terms of the
 number of comparisons? What is it in terms of the number of flips? Add your
 answer to this markdown file.
 
+The outer loop runs n-1 times, which is an element of $\Theta$(n). The inner loop makes j comparisons each pass, where j is the size of the unsorted array, so j = n - 1 for the first pass, j = n - 2 for the second pass. Thus we can think of the amount of work done by the inner loop as an element of $\Theta$(n). Multiplying these two gives us a total time complexity of $\Theta$(n<sup>2</sup>).
+
+Flips can be thought of in a very similar way. Each element we sort can take 0, 1, or 2 flips depending on its position. Doing this for every element gives us an asymptotic complexity for number of flips of $\Theta$(n), linear to number of elements. Within each flip we return to our trusty j, with j being the number of unsorted elements, n for first pass, n-1 for second pass, once again simplifying to $\Theta$(n). Multiplying these two once again gives us $\Theta$(n<sup>2</sup>)
```

Comment:
Are you really doing a flip for each comparison?

Reply:
Not every comparison results in a flip. If an element is already in its correct position no flips are performed. If the largest non sorted element is already at the front of the array, one flip is performed. If the element is somewhere in the middle, two flips are performed. For every element (every iteration of the outer loop) we perform a max of two flips, meaning that asymptotic analysis of the number of flips would have it linear to the number of elements (n)

Reply:
So is the number of flips linear or quadratic?

Reply:
The number of flips is linear, the runtime evaluation using number of flips would still be quadratic (if that makes sense, I’m not getting the feeling I misunderstood the questions and you just wanted an analysis of number of flips, not runtime analyzed using flips)

Reply:
It's just the number of flips.